src/IdePurescript/Atom/Config.purs

```diff
@@ -17,7 +17,7 @@ import Node.Platform (Platform(Win32))
 import Node.Process as P
 
 defaultSrcGlob :: Array String
-defaultSrcGlob = ["src/**/*.purs", "bower_components/**/*.purs"]
+defaultSrcGlob = ["src/**/*.purs", "bower_components/**/*.purs", ".psc-package/**/*.purs"]
```

Comment:
One of the problems with this Glob is that it also picks up test files for dependencies, as well as multiple different package sets.

I'm using this Glob for my current project `".psc-package/psc-0.11.6/*/*/src/**/*.purs"`, but of course that only works if you're using the psc-0.11.6 package set.

I don't think this can just be a static string, since you need to analyze the current projects psc-package.json file.

Reply:
I agree, see comments on this issue: https://github.com/nwolverson/atom-ide-purescript/issues/170. You can of course use this glob in your own settings if it works for you.

Also and unrelated to this change, per @kRITZCREEK's point it looks like those bower paths are wrong too :|

Reply:
@nwolverson Oh, yep. The globs we usually use:

`"src/**/*.purs" "bower_components/purescript-*/src/**/*.purs"`

Reply:
Not that there are any packages that are not purescript left in bower_components usually, so that part can go, but the `src/` bit is relevant.

Reply:
Sorry, I have not tested this case carefully. I just try to use psc-package, but hyperclick can  not work. I add psc-package location, hyperclick works again. So I create this pull request.

Sorry. @nwolverson 
Thank you  @kRITZCREEK

Reply:
So the right way I think atom should check the project directory:

* if exist `bower_components`, then glob `"src/**/*.purs", "bower_components/purescript-*/src/**/*.purs"`
* elseif exist `.psc-package`, then check `psc-package.json` ,  to confirm which set  is  chosen now, then glob `"src/**/*.purs", ".psc-package/$SETNAME/*/*/src/**/*.purs"`

Reply:
I think I'd always prefer psc-package where present - I was thinking "if `psc-package.json` exists then add the output globs from `psc-package sources`".